akka-remote/src/main/resources/reference.conf

```diff
@@ -93,6 +93,28 @@ akka {
       "akka.actor.ActorKilledException" = akka-misc
       "akka.actor.InvalidActorNameException" = akka-misc
       "akka.actor.InvalidMessageException" = akka-misc
+
+      "akka.actor.LocalScope$" = akka-misc
+      "akka.remote.RemoteScope" = akka-misc
+
+      "com.typesafe.config.impl.SimpleConfig" = akka-misc
```

Comment:
Yes, that was the problem I had, it chose `Serializable` over this. Couldn't find any other implementations.